### PR TITLE
Add Glaxnimate

### PR DESCRIPTION
**Pull request by mbasaglia**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(I added filesystem access as the user might want to open/save files)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.mattbas.Glaxnimate.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+runtime: org.kde.Platform
+runtime-version: "6.3"
+sdk: org.kde.Sdk
+sdk-version: "6.3"
+command: glaxnimate
+finish-args:
+    - --share=ipc
+    - --share=network
+    - --socket=x11
+    - --socket=wayland
```
Comment (barthalion) [inline org.mattbas.Glaxnimate.yml:11]:
```suggestion
    - --socket=fallback-x11
    - --socket=wayland
```

File: org.mattbas.Glaxnimate.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        builddir: true
+        config-opts:
+            - -DCMAKE_BUILD_TYPE=Release
+            - -DGLAXNIMATE_SYSTEM_POTRACE=OFF
+        sources:
+            -
+                type: git
+                url: https://gitlab.com/mattbas/glaxnimate.git
+                tag: 0.5.1+flatpak
+                commit: a5ddc69671f18c5f18f6bbb1ae27924c596a06bd
```
Comment (barthalion) [inline org.mattbas.Glaxnimate.yml:27]: if it makes your life simpler, it's fine to use only `commit` here.

File: org.mattbas.Glaxnimate.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    - --share=ipc
+    - --share=network
+    - --socket=x11
+    - --socket=wayland
+    - --filesystem=host
+    - --device=dri
+modules:
+    -
+        name: glaxnimate
+        buildsystem: cmake
```
Comment (barthalion) [inline org.mattbas.Glaxnimate.yml:17]:
```suggestion
        buildsystem: cmake-ninja
```

File: org.mattbas.Glaxnimate.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+runtime-version: "6.3"
+sdk: org.kde.Sdk
+sdk-version: "6.3"
+command: glaxnimate
+finish-args:
+    - --share=ipc
+    - --share=network
+    - --socket=x11
+    - --socket=wayland
+    - --filesystem=host
```
Comment (barthalion) [inline org.mattbas.Glaxnimate.yml:12]: Does it use file portal? We discourage direct filesystem access like this if file picker can do the job.

**Comment by barthalion** [comment]
bot, build org.mattbas.Glaxnimate

File: org.mattbas.Glaxnimate.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    - --device=dri
+modules:
+    -
+        name: glaxnimate
+        buildsystem: cmake
+        builddir: true
+        config-opts:
+            - -DCMAKE_BUILD_TYPE=Release
```
Comment (hfiguiere) [inline org.mattbas.Glaxnimate.yml:20]:
```suggestion
            - -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

File: org.mattbas.Glaxnimate.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
+app-id: org.mattbas.Glaxnimate
+runtime: org.kde.Platform
+runtime-version: "6.3"
```
Comment (hfiguiere) [inline org.mattbas.Glaxnimate.yml:3]: the README only mention qt5. Are you sure you want Qt6?

**Comment by hfiguiere** [comment]
```
builddir/files/share/appdata/org.mattbas.Glaxnimate.appdata.xml: FAILED:
• tag-missing           : <release> required
Validation of files failed
```

File: org.mattbas.Glaxnimate.yml (hunk 6 above)
Comment (mbasaglia) [reply org.mattbas.Glaxnimate.yml:3]: Yeah, works with both. I ported everything to work on Qt6 as of the latest release

File: org.mattbas.Glaxnimate.yml (hunk 4 above)
Comment (mbasaglia) [reply org.mattbas.Glaxnimate.yml:12]: It can load files based on IPC commands, would the portal work in that case?

Comment (barthalion) [reply org.mattbas.Glaxnimate.yml:12]: I assume it wouldn't.

**Comment by barthalion** [comment]
bot, build org.mattbas.Glaxnimate

**Pull request closed without merging**